card/src/main/java/com/adyen/checkout/card/internal/ui/view/CardNumberField.kt

```diff
@@ -115,10 +107,7 @@ private fun CardNumberInputField(
         shouldFocus = cardNumberState.isFocused,
         trailingIcon = {
             CheckoutNetworkLogo(
-                modifier = Modifier
-                    .height(16.dp)
-                    .width(24.dp)
-                    .clip(RoundedCornerShape(Dimensions.CornerRadius)),
+                modifier = Modifier.size(width = 24.dp, height = 16.dp),
```

Comment:
I think we will need this size in more and more places. Should we define a constant?